service/src/main/java/cz/muni/fi/pv254/parsing/App.java

```diff
@@ -374,6 +468,8 @@ public String downloadUserName(long userId) {
                 Element body = doc.body();
                 Elements nieco = body.getElementsByAttributeValue("class","actual_persona_name");
                 if (nieco.isEmpty()) {
+                    // TODO tu to niekedy pada ze nie je strana dostupna
+                    // TODO aj tu to treba zopakovat {tries}
                     throw new IllegalArgumentException("User Name not found for id "+Long.toString(userId));
```

Comment:
niekedy to vrati nepaltnu stranku ze je pretazeny alebo nedosutpny tak vybehne tato exception a prestane to spracovavat lebo dostane null name
vyriesim to inokedy